docs/guides/Redux-Integration.md

```diff
@@ -1,7 +1,13 @@
 # Redux Integration
 
-To handle your app's navigation state in redux, you can pass your own `navigation` prop to a navigator. Your navigation prop must provide the current state, as well as access to a dispatcher to handle navigation options.
+### Roadmap for redux integration
+1. To handle your app's navigation state in redux, you can pass your own `navigation` prop to a navigator.
 
+2. Once you pass your own navigation prop to the navigator, the default [`navigation`](https://reactnavigation.org/docs/navigators/navigation-prop) prop gets destroyed, you need to pass your own `navigation` prop. You will most probably pass the `navigation` prop's properties that you want to access. Normally  [`state`](https://reactnavigation.org/docs/navigators/navigation-prop#state-The-screen's-current-stateroute) and [`dispatch`](https://reactnavigation.org/docs/navigators/navigation-prop#dispatch-Send-an-action-to-the-router) properties are passed to the navigator. You will learn how to pass those properties further in this guide. Suppose if you didn't pass `dispatch`  to the navigator and only passes `state` than you can't access `dispatch` further in your Components. Again this is because the default `navigation` prop was destroyed which includes the defualt `dispatch` method of react-navigation.
```

Comment:
The first phrase "Once you pass your own navigation prop to the navigator, the default [`navigation`](https://reactnavigation.org/docs/navigators/navigation-prop) prop gets destroyed, you need to pass your own `navigation` prop." has a repetition; this way would be better: "Once you pass your own navigation prop to the navigator, the default [`navigation`](https://reactnavigation.org/docs/navigators/navigation-prop) prop gets destroyed."

Reply:
And I'd rewrite this part: "Suppose if you didn't pass `dispatch`  to the navigator and only passes `state` than you can't access `dispatch` further in your Components. Again this is because the default `navigation` prop was destroyed which includes the defualt `dispatch` method of react-navigation."

Reply:
Yeah, thanks for the input. I will incorporate the changes 👍 .